### PR TITLE
Add documentation about bulk close/open endpoints

### PR DESCRIPTION
**Pull request by ivaylomitrev**

### CODE REVIEW

File: version1/04 - Web services.md [hunk 1]
```diff
@@ -858,3 +859,66 @@ Both **groupId** and **fieldId** are strings comprised of lower-case letters, di
...
+**Request**
+
+``` text
+GET /rms/api/public/noark5/v1/bulk/close HTTP/1.1
+Authorization: Bearer ACCESS_TOKEN
+Content-Type: application/json
+
+{
+  "type": string,
+  "id": number
```
Comment (teodoradobreva) [inline version1/04 - Web services.md]: Should be a string.

File: version1/04 - Web services.md [hunk 2]
```diff
@@ -858,3 +859,66 @@ Both **groupId** and **fieldId** are strings comprised of lower-case letters, di
...
+**Request**
+
+``` text
+GET /rms/api/public/noark5/v1/bulk/open HTTP/1.1
+Authorization: Bearer ACCESS_TOKEN
+Content-Type: application/json
+
+{
+  "type": string,
+  "id": number
```
Comment (teodoradobreva) [inline version1/04 - Web services.md]: Should be a string.

File: version1/04 - Web services.md (hunk 1 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Fixed.

File: version1/04 - Web services.md (hunk 2 above)
Comment (ivaylomitrev) [reply version1/04 - Web services.md]: Fixed.

**Comment by ivaylomitrev** [comment]
Rebased on the latest develop.

**Review by teodoradobreva — APPROVED** [review]

**Review by orlinDocumaster — APPROVED** [review]

**Comment by ivaylomitrev** [comment]
Simply rebased on the latest develop and will merge the PR now.

**Comment by ivaylomitrev** [comment]
Also squashed.

**Pull request merged**